content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/profiles/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Profiles"
+type: "article"
+description: "An overview of Chainguard Enforce's observer and enforcer profiles"
+date: 2023-03-24T15:56:52-07:00
+lastmod: 2023-03-24T15:56:52-07:00
+draft: false
+tags: ["ENFORCE", "CHAINCTL", "PRODUCT", "OVERVIEW"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 90
+toc: true
+---
+
+When you install Chainguard Enforce, you have the option to specify a *profile* that will determine what permissions Chainguard Enforce has over your cluster. Currently, Enforce offers two specialized profiles: `enforcer` and `observer`.
+
+The `observer` profile has less expansive permissions, making it useful for running Enforce in a sort of "read-only" mode. The `enforcer` profile, on the other hand, has broader permissions that enable you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement.
```

Comment:
I'm not sure what "sort of 'read-only'" means here, but is there more precise language that can be used, or can you say it is "read-only" or "virtually read-only"?

Also subject-verb agreement here: "enforce profile ... enables you..."

```suggestion
The `observer` profile has less expansive permissions, making it useful for running Enforce in a sort of "read-only" mode. The `enforcer` profile, on the other hand, has broader permissions that enables you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement.
```

Reply:
Definitely with you on the "read-only" part, though I think the subj/verb agreement part is okay as it is. I did try changing it but I feel like this wording is as clear as I can make it.

At least how i read it, the broader permissions are what are doing the enabling here, so "X has broader permissions that enable you to...." scans. LMK if you still disagree and I'll try some more wordsmithing.

Reply:
What about "The broader permissions of the `enforcer` profile enables you to benefit from all of Chainguard Enforce's features, including policy distribution and enforcement."

Reply:
Works for me :+1: